code/src/Components/Temperature.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+  const Temperature= ({temperature, setTemperature, handlePreviousQuestionClick, handleNextQuestionClick, counter }) => {
+
+  return (
+  <>
+    <div className="questionWrapper">
+      <p> What temperature do you prefer on your hollyday?</p>
+        <p className='degrees'>{temperature}°</p>
+    </div>
+
+    <div className="temperatureWrapper">
+      <form>
+      <input
+        value= {temperature}
+        className="slider"
```

Comment:
So cool that you used a slider! I am really impressed how you work with your project. Going to keep this for inspiration :)